build.js

```diff
@@ -155,7 +155,7 @@ const autoninja =
   currentPlatform() == "windows" ? "autoninja.bat" : "autoninja";
 // make the windows build verbose so we can see what's going on
 const platformAutoNinjaArgs = currentPlatform() == "windows" ? ["-v"] : [];
-spawnChecked(autoninja, [...platformAutoNinjaArgs, "-C", outdir, "chrome"], {
+spawnChecked(autoninja, ["-v", ...platformAutoNinjaArgs, "-C", outdir, "chrome"], {
```

Comment:
?

Reply:
That was an attempt at [RUN-3228](https://linear.app/replay/issue/RUN-3228/compiler-errors-dont-show-in-ci), but it did not work

Reply:
Sorry: is this supposed to be in this PR?  How does it relate?  If it does not work, why is it here?

Reply:
I removed it, but... the push didn't work 😠... Have to do it tmrw...

Reply:
Why, you little...
![image](https://github.com/replayio/chromium/assets/282899/04290b49-3b36-49a0-b244-ce4ba769c5dd)